protected/humhub/modules/admin/views/information/database.php

```diff
@@ -31,6 +48,7 @@
 
 <div class="well">
```

Comment:
Hmm, instead of preparing the button on the controller side, I would prefer to add some vars, to get a simple and readable view e.g. 

```php
<?php if ($newMigrationsPerformed): ?>
       Database Migration Result:
      <pre>
        <?= $migrationOutput ?>
      </pre>
      <?= Html::a('Update Database' , ['/admin/information/database'], ['class' => 'btn btn-default']); ?>
<?php elseif ($newMigrationsAvailable): ?>
      <pre>
        <?= $migrationOutput ?>
      </pre>
      <?= Html::a('Refresh' , ['/admin/information/database', 'migrate' => 1], ['class' => 'btn btn-default']); ?>
<?php else: ?>
      <!-- Show your database is up2date message -->
<?php endif; ?>
```